### PR TITLE
Raises exception on duplicate key or duplicate value

### PR DESCRIPTION
**Pull request by laertispappas**
Do not allow for duplicate keys || duplicate values.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-### 0.5.1 (Next)
+### 0.6.1 (Next)
 
 * Your contribition here.
 
+### 0.6.0 (27/11/2016)
```
Comment (dblock) [inline CHANGELOG.md]: Since you're not making a release, undo this, leave versions alone.

**Comment by dblock** [comment]
This is great. You can just remove old rubies and make the build green.

File: CHANGELOG.md (hunk 1 above)
Comment (laertispappas) [inline CHANGELOG.md]: Sure.

**Comment by laertispappas** [comment]
Do you know why the build has failed?

**Comment by laertispappas** [comment]
Not sure why rb-x is failing

**Comment by dblock** [comment]
Just remove rbx-2, it's a PITA. Sorry about that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
-### 0.5.1 (Next)
+### 0.6.1 (Next)
```
Comment (dblock) [inline CHANGELOG.md]: Don't do this, leave versions alone here.

Comment (laertispappas) [inline CHANGELOG.md]: Sure I will. Sorry for the misunderstanding. I've just started the open source involvement.  Thanks for your feedback.

**Comment by laertispappas** [comment]
Removed rbx-2 from travis.yml. By the way what does PITA mean?

**Comment by dblock** [comment]
PITA - Pain in the Ass :)

**Comment by dblock** [comment]
Merged via https://github.com/dblock/ruby-enum/commit/4b7c6eb868d08c2c3b126a96527eeef0775fcd8f.

**Comment by dblock** [comment]
FYI I released 0.6.0 with this.

**Pull request closed without merging**